pan3d/dataset_builder.py

```diff
@@ -27,7 +29,24 @@
 
 @TrameApp()
 class DatasetBuilder:
-    def __init__(self, server=None, dataset_path=None, state=None, pangeo=False):
+    """Manage data slicing, mesh creation, and rendering for a target N-D dataset."""
+
+    def __init__(
+        self,
+        server: trame_server.core.Server = None,
+        dataset_path: str = None,
+        state: dict = None,
+        pangeo: bool = False,
+    ) -> None:
+        """Create an instance of the DatasetBuilder class.
+
+        Parameters:
+            server: Trame server instance.
+            dataset_path: A path or URL referencing a dataset readable by xarray.open_dataset()
+            state:  A dictionary of initial state values.
+            pangeo: If true, use a list of example datasets from Pangeo Forge (examples/pangeo_catalog.json).
+        """
+
```

Comment:
The lines below could be replaced with

```python
server = get_server(server, client_type="vue3")
```

Reply:
Changed in [aa5dbc8](https://github.com/Kitware/pan3d/pull/47/commits/aa5dbc87edb31c90529218b433aa876d09299801)